plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1200,8 +1197,6 @@ public void testJoinPushdown()
                 "nation_lowercase",
                 "AS SELECT nationkey, lower(name) name, regionkey FROM nation")) {
             for (JoinOperator joinOperator : JoinOperator.values()) {
-                log.info("Testing joinOperator=%s", joinOperator);
```

Comment:
You can replace it with a `describedAs(...)` in the assertion that indicates which join operator is being evaluated. That will be included in the failure message.